### ISSUE TITLE
Improve methbase description page

### ISSUE THREAD

**Issue by jqujqu** [smithlabcode/methpipe#84]
@guilhermesena and @timydaley  will take a look at the description page and comment on what's missing.  Others should follow up keep the discussion going.

**Comment by guilhermesena** [comment]
Ok, here are my observations.

I used [this project] (https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=451133223_oiNbGhSxFdCdpp68aWAXIqqYzFLk&c=chr21&g=hub_8415_Hodges_Human_2011) as a reference. Overall I think it's very synthetic and shows what has to be shown. There are some minor details that weren't clear to me (maybe for lack of experience in sequenced data).

**Description**
 - The second column doesn't have a label. Maybe "Sample description" or something? 
 - What's the unit for coverage? Is it something like the average number of times a nucleotide has been mapped? I couldn't find how this was calculated within the page or if this is information that is inferred from the sequencing.
 - Maybe larger spacing between the last columns (HMRs, AMRs and PMDs)? 

**Methods**
 - Identifying HyperMRs, PMDs and AMRs: I get that HMRs can be inferred using HMMs and I guess the same logic applies for HyperMRs, but what about PMDs and AMRs? How are they being identified? Is the PMD state something like "close to 0.5"? 

That's all I can think about for now. I'll read it again and edit this post if something else is unclear.

**Comment by jqujqu** [comment]
Update the section **Mapping bisulfite treated reads** section  with information about `walt` too. 

Add reference to Fang's paper either in the **Identifying allele-specific methylated regions** or in the **References**

**Comment by egor-dolzhenko** [comment]
Perhaps those of us who had significant interactions with MethBase users could tell us about how people use the database? This could help us to make the description page more useful.

I typically use MethBase to visually compare methylomes in a specific locus. So it would be useful to know if some "global" properties of any of the methylomes that I am comparing are off. For example, if one of the methylomes has abnormally many HMRs or the HMRs are extra wide or if the average methylation level is very low then I would not be very surprised to see regional differences. If, on the other hand, all genome-wide stats of all methylomes match closely, I would be more confident that the region I am looking at is special in some way.

To summarize, it would be cool to have some "normality" measures (which could be defined based on all methylomes or the closest group of reference methylomes) displayed on the description page.
